internal/Session/Timing/Math.hs

```diff
@@ -12,7 +12,7 @@ import Time
 -- | Calculate the next point in time where the given session
 --   will expire assuming that it sees no activity until then
 --
--- Returns 'Nothing' iff the settings do not specify any timeout limits.
+-- Returns 'Nothing' if the settings do not specify any timeout limits.
```

Comment:
"iff" is an abbreviation of "if any only if," I probably shouldn't have abbreviated it.